src/index.template.html

```
@@ -5,6 +5,12 @@
     <meta name="description" content="<%= htmlWebpackPlugin.options.productDescription %>" />
     <meta name="format-detection" content="telephone=no" />
     <meta name="msapplication-tap-highlight" content="no" />
+    <% if (ctx.prod) { %>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)